data/migrations/20190515102553_FriendJoinTable.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('friends', function(friends){
+        friends.increments();
+        friends.string('friend_name',128).notNullable();
```

Comment:
We only need reference keys from other tables (no need for strings). Here we only need user_id and friend_id (both of them from the users table).

Reply:
here we don't need friends name